client/components/pages/RegisterPage/RegisterPage.js

```diff
@@ -13,7 +13,7 @@ export default function RegisterPage() {
 
   useEffect(() => {
     if (!R.isEmpty(user)) {
-      dispatch(push("/home"));
+      dispatch(push("/register"));
```

Comment:
why?

Reply:
I changed it and went back to what it was